deploy/crds/managed_v1alpha1_grouppermission_crd.yaml

```diff
@@ -32,13 +32,7 @@ spec:
             clusterPermissions:
               description: List of permissions applied at Cluster scope
               items:
-                properties:
-                  clusterRoleName:
-                    description: Name of ClusterRole to bind to the Group as a ClusterRoleBinding
-                    type: string
-                required:
-                - clusterRoleName
-                type: object
+                type: string
```

Comment:
dumb question, but tis this right?  This very well could be because I don't understand this, but it looks funny.

I see this as the final file:

```
        spec:
          properties:
            clusterPermissions:
              description: List of permissions applied at Cluster scope
              items:
                type: string
              type: array
```

Reply:
This means the elements in the array are type string.

Reply:
ignore me, thanks!